unity-environment/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAcademy.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WalkerAcademy : Academy
+{
+    public override void InitializeAcademy()
+    {
+        Monitor.verticalOffset = 1f;
+        Physics.defaultSolverIterations = 12;
```

Comment:
Can you comment on why the values are increased from the default.

Reply:
@Hunter-Unity Could you comment on these?